lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -602,51 +601,6 @@ def create_relation_methods
           end
         end
 
-        #: -> void
-        def create_association_relation_methods
```

Comment:
I would still like to keep this method, even if it is empty now, since there could be methods on association relation that don't exist on normal relations that we might want to cater for.